independent-publisher-connectors/EONET by NASA/apiDefinition.swagger.json

```diff
@@ -0,0 +1,977 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "EONET by NASA",
+    "description": "The Earth Observatory Natural Event Tracker (EONET) is a prototype web service from NASA with the goal of:\n\nProviding a curated source of continuously updated natural event metadata.\nProviding a service that links those natural events to thematically-related web service-enabled image sources (e.g., via WMS, WMTS, etc.).",

```

Comment:
@troystaylor Can you please remove the HTML tags in the value of connector's description.

Reply:
Done